ui/src/containers/InventoryLayout.js

```diff
@@ -140,6 +171,29 @@ const InventoryLayout = (props) => {
             </TableBody>
           </Table>
         </TableContainer>
+        <InventoryFormModal
+          title='Create'
+          formName='inventoryCreate'
+
+          isDialogOpen={isCreateOpen}
+          handleDialog={toggleModals}
+          handleProduct={createInventories}
+          initialValues={{}}
+        />
+        <InventoryFormModal
```

Comment:
While an `Edit` and a `Delete` form modal are eventually implemented, it avoids the ticket's purpose, which distracts your intended work and objective. There will be scenarios where preemptive implementation is incorrect or it's not even needed anymore. You can leave the `Edit` and `Delete` form modals here, but do not continue working on them until you're tasked to do so.